tests/sudorule/test_sudorule.yml

```diff
@@ -53,11 +87,87 @@
     register: result
     failed_when: result.changed
 
-  - name: Ensure sudorule is present, runAsUserCategory.
+  - name: Add user01 to the list of users sudorule execute as.
```

Comment:
All adds and removes here should be ensure presence/absence of ...

Reply:
fixed.